src/pages/programDisaggregations/form/ProgramIndicatorMappingSection.tsx

```diff
@@ -184,6 +184,10 @@ const ProgramIndicatorCard = ({
     )
 }
 
+const addDefaultCategoryComboTransform = (results: any[]) => {
```

Comment:
🙈 I had trouble with getting the type definition to work with the enum (`DISAGGREGATION`, `ATTRIBUTE`), so would appreciate some troubleshooting help here 🙏